CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Changed
 
 - Base `setup_duration` within optimizer-ortools on `point_id` instead of `matrix_index` [#444] (https://github.com/Mapotempo/optimizer-api/pull/444)
```

Comment:
```suggestion
- Base `setup_duration` within optimizer-ortools on `point_id` instead of `matrix_index` [#444] (https://github.com/Mapotempo/optimizer-api/pull/444) [#446](https://github.com/Mapotempo/optimizer-api/pull/446]
```